modules/codegen/src/main/scala/com/twilio/guardrail/generators/Http4sServerGenerator.scala

```diff
@@ -713,6 +714,24 @@ object Http4sServerGenerator {
       decoders.distinctBy(_._1.toString()).map(_._2) ++ matchers
     }
 
+    /**
+      * It's not possible to use backticks inside pattern matching as it has different semantics: backticks inside match
+      * are just references to an already existing bindings.
+      */
+    def prepareParameters[F[_]: Traverse](parameters: F[ScalaParameter[ScalaLanguage]]): Target[F[ScalaParameter[ScalaLanguage]]] =
+      Traverse[F].traverse(parameters) { param =>
+        if (param.paramName.syntax == param.paramName.value) {
+          Target.pure(param)
+        } else {
+          // let's try to prefix it with underscore and see if it helps
```

Comment:
This'll potentially generate conflicts as well. I'm going to rewrite this to prefix _all_ parameters if _any_ need to be prefixed.

Reply:
updated